src/window.cc

```diff
@@ -24,11 +21,13 @@ namespace sigc {
 #endif
 }
 
-Window::Window() : compiling(false), debugging(false) {
+Window::Window() : notebook(Notebook::get()) {
   JDEBUG("start");
```

Comment:
It's stored for convenience, notebook is used quite a few places in window.cc and writing Notebook::get(). instead of notebook. looked ugly. But can change it to Notebook::get() if you rather like that.